### ISSUE TITLE
Dereferenceable IRIs and content negotiation

### ISSUE THREAD

**Issue by andrefs** [yago-naga/yago4#13]
Hello,

(Not sure whether this is the best place to raise this issue, but I couldn't find any other contact information on YAGO's homepage.)

When dereferencing YAGO's IRIs I am obtaining an HTML response. I'm trying to obtain an RDF response. 

I've tried sending an `Accept` header or looking for a `<link rel="alternate" type="some rdf media type" href="...">` in the HTML but neither approach is working.

Am I missing something or is YAGO incapable of dereferencing IRIs in RDF format?

Thanks

**Comment by Tpt** [comment]
Hi! It's indeed a great place to raise issues about YAGO 4.

Thank you for reporting this problem. Content negotiation should work but it seems it does not indeed.
`curl -H "accept: text/turtle" https://yago-knowledge.org/resource/Julius_Caesar` returns me HTML and not Turtle.

@mclegrand Have you tweaked the nginx configuration after my departure?

**Comment by mclegrand** [comment]
> @mclegrand Have you tweaked the nginx configuration after my departure?

No, I did not touch anything, The main event that happened recently was a power outage affecting the server room a few weeks ago but it should just have restarted the server normally

**Comment by Tpt** [comment]
@mclegrand Thank you! Sorry for disturbing you.

I am not sure what cause this problem. YAGO website should be able to properly handle content negotiation using [this code](https://github.com/yago-naga/yago-website/blob/master/resource.php). I do not have access to the server anymore so I am not able to have a look at its configuration or logs to debug this problem. Sorry.

**Comment by almereyda** [comment]
This seems fixed now, and could be closed:

```shell
$ curl -s -H "accept: text/turtle" https://yago-knowledge.org/resource/Julius_Caesar | head -n 12
@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
@prefix sesame: <http://www.openrdf.org/schema/sesame#> .
@prefix owl: <http://www.w3.org/2002/07/owl#> .
@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
@prefix fn: <http://www.w3.org/2005/xpath-functions#> .
@prefix foaf: <http://xmlns.com/foaf/0.1/> .
@prefix dc: <http://purl.org/dc/elements/1.1/> .
@prefix hint: <http://www.bigdata.com/queryHints#> .
@prefix bd: <http://www.bigdata.com/rdf#> .
@prefix bds: <http://www.bigdata.com/rdf/search#> .

```
